components/Typography/src/paragraph.scss

```diff
@@ -40,3 +40,25 @@
 .denhaag-list__wrapper + .utrecht-paragraph--lead {
   --utrecht-paragraph-margin-block-start: var(--utrecht-paragraph-lead-list-margin-block-start, inherit);
 }
+
+@media (min-width: 768px) {
+  *:not(.utrecht-paragraph):not(.utrecht-heading-1):not(.utrecht-heading-2):not(.utrecht-heading-3):not(.utrecht-heading-4):not(.utrecht-heading-5)
```

Comment:
Not sure, but doesn't `:not([class^='utrecht-heading']):not([class*=' utrecht-heading'])` also work? If so, this CSS gets a lot shorter